spec/directory_importer_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe IssueExporting::DirectoryImporter do
+  describe '#import' do
+    it 'gets all the json files in a dir' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.